tidy3d/components/data/sim_data.py

```diff
@@ -792,6 +792,28 @@ def plot_scalar_array(
 
         return ax
 
+    def to_mat(self, fname):
+        """Output the SimulationData object as .mat MATLAB file
+
+        Parameters
+        ----------
+        fname : str
+            Full path to the output file. Should include .mat file extension
+
+        Example
+        -------
+        >>> simData.to_mat('/path/to/file/data.mat') # doctest: +SKIP
+        """
+        # Get SimData object as dictionary
```

Comment:
let's throw a check here to make sure the extension of `fname` is `.mat`. Otherwise the user probably doesn't want this method and they should use `to_file()`.